src/docfx/docset/Document.cs

```diff
@@ -471,7 +471,8 @@ private static bool IsValidRelativePath(string path)
             // get set path from site path
             // site path doesn't contain version info according to the output spec
             var sitePathWithoutExtension = Path.Combine(Path.GetDirectoryName(SitePath), Path.GetFileNameWithoutExtension(SitePath));
-            var sitePath = PathUtility.NormalizeFile(Path.GetRelativePath(Docset.Config.DocumentId.SiteBasePath, sitePathWithoutExtension));
+            var siteBasePath = Docset.Config.Output.LowerCaseUrl ? Docset.Config.DocumentId.SiteBasePath.ToLowerInvariant() : Docset.Config.DocumentId.SiteBasePath;
```

Comment:
Is there a bug for this one? suggest not change this in this PR.

Reply:
this bug is found in my test, so i need fix it to make my test pass in linux.